db/sql/src/channels.rs

```diff
@@ -261,6 +266,24 @@ impl HoprDbChannelOperations for HoprDb {
             .await
     }
 
+    async fn stream_active_channels<'a>(&'a self) -> Result<BoxStream<'a, Result<ChannelEntry>>> {
+        Ok(Channel::find()
+            .filter(
+                channel::Column::Status
+                    .eq(i8::from(ChannelStatus::Open))
+                    .or(channel::Column::Status
+                        .eq(i8::from(ChannelStatus::PendingToClose(
+                            hopr_platform::time::native::current_time(), // irrelevant
```

Comment:
Why are we considering pending to close channels here too?

Reply:
Yes, those that have still active grace period are synced into the graph, but they are not considered for path planning. It is just for tracking state changes for strategies.